packages/core/package.json

```diff
@@ -8,6 +8,7 @@
     "@maticnetwork/fx-portal": "^1.0.4",
     "@openzeppelin/contracts": "4.4.2",
     "@uma/common": "^2.28.4",
+    "@uma/merkle-distributor": "^1.3.38",
```

Comment:
Could this be a dev dependency? Isn't it only used in tests? Or is this a contract import?

Reply:
that's true! only used in tests. will move it.